### ISSUE TITLE
Error in version 1.0.1: DropDownView.OnSelectionListener does not exist

### ISSUE THREAD

**Issue by lukascarrilho** [siralam/DropDownView#3]
In the example bellow, DropDownView.OnSelectionListener exist in version 1.0.0 but not in version 1.0.1.

dropDownView.setOnSelectionListener(new DropDownView.OnSelectionListener() {
            @Override
            dropDownView.onItemSelected(DropDownView view, int position) {
                //Do something with the selected position
            }
        });

**Comment by siralam** [comment]
Ya, thats why there is 1.0.1

**Comment by siralam** [comment]
You are right, just realized that my previous testing on a Java class did not use v1.0.1....

In the meantime, you can use the below code instead:
```
        dropDownView.setOnSelectionListener(new Function2<View, Integer, Unit>() {
            @Override
            public Unit invoke(View view, Integer integer) {
                //Do something you want
                return null;
            }
        });
```

I am trying to find how to expose the Kotlin expression variable to something named in Java.

**Comment by siralam** [comment]
@lukascarrilho Solved in v1.0.2. Please update and use 

```
dropDownView.setOnSelectionListener(new OnDropDownSelectionListener() {
@override
public void onItemSelected(DropDownView view, int position) {
//Do something with the selected position
}
});
```

Thank you for reporting this issue!
